devs/docs/release.rst

```diff
@@ -105,5 +105,21 @@ archived.
 You can archive releases by selecting *Edit*, unselecting the *Active*
 checkbox, and then saving.
 
+Release Notes 
+=======================
+
+Once a release is done, we create a release note, which is
+
+- Added to the documentation
+- Sent out via mail as testing release note
+
+Keep each release note simple and provide the right level of information. Everyone shall be able to understand the impact of the changes without the need to go into the depth of tech. 
```

Comment:
I think we should include these hints as invisible comments in `https://github.com/crate/crate/blob/master/blackbox/docs/appendices/release-notes/unreleased.rst`. So it is visible each time somebody makes a change to the file and is reminded of these things.

We tend to have a documentation discoverability problem instead of there not being enough documentation.